pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2994,6 +2998,9 @@ def predict_model(
                 X_test_ = None
             y_test_ = self.y_test.copy()
 
+            # y_train for finalized model is different from self.y_train
+            # Hence, better to get this from the estimator directly.
+            y_train = estimator_._y
```

Comment:
Do we store targets in estimators? Is that something we have introduced, or is that an sktime thing?

Reply:
sktime does this for time series. It is needed for making predictions since these depend on past values.